test/kubemark/iks/config.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+
+# Copyright 2018 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Cloud information
+CLOUD_PROVIDER="${CLOUD_PROVIDER:-iks}"
```

Comment:
Do you really need to redefine this variable here? All the right bash exports should automatically if you set the env var here - https://github.com/kubernetes/kubernetes/blob/master/test/kubemark/cloud-provider-config.sh#L17 (the purpose of that file is pretty much this :)